src/pages/Validator/DetailCard.tsx

```diff
@@ -0,0 +1,114 @@
+import {Box, Skeleton} from "@mui/material";
+import * as React from "react";
+import HashButton from "../../components/HashButton";
+import ContentBox from "../../components/IndividualPageContent/ContentBox";
+import ContentRow from "../../components/IndividualPageContent/ContentRow";
+import {HashType} from "../../components/HashButton";
+import RewardsPerformanceTooltip from "../Validators/Components/RewardsPerformanceTooltip";
+import LastEpochPerformanceTooltip from "../Validators/Components/LastEpochPerformanceTooltip";
+import {prettifyTimestamp} from "../utils";
+import {Types} from "aptos";
+import {
+  useGetMainnetValidators,
+  useGetValidatorToOperator,
+} from "../../api/hooks/useGetValidatorSet";
+import {useGetAccountResource} from "../../api/hooks/useGetAccountResource";
+import {ValidatorCardDetailName, validatorCardDetailType} from "./Constants";
+import {useEffect, useState} from "react";
+
+type ValidatorDetailProps = {
+  address: Types.Address;
+};
+
+export default function ValidatorDetailCard({address}: ValidatorDetailProps) {
```

Comment:
It's very nice to explore different coding styles! Good job! However, for consistency across the code base, I would recommend implementing the content card/tab in similar style. Take a look at `UserTransactionOverviewTab` and places that uses `<ContentRow .../>`.

Reply:
ofc! initially was thinking how to write shorter code with concise render function but consistency def matters. will change it back!